planning/behavior_velocity_planner/autoware_behavior_velocity_intersection_module/src/scene_intersection.cpp

```diff
@@ -392,9 +392,10 @@ DecisionResult IntersectionModule::modifyPathVelocityDetail(
     if (!has_traffic_light_) {
       if (fromEgoDist(occlusion_wo_tl_pass_judge_line_idx) < 0) {
         if (has_collision) {
-          const auto closest_idx = intersection_stoplines.closest_idx;
+          const auto closest_stopline_idx = intersection_stoplines.closest_idx;
           const std::string evasive_diag = generateEgoRiskEvasiveDiagnosis(
-            *path, closest_idx, time_distance_array, too_late_detect_objects, misjudge_objects);
+            *path, closest_stopline_idx, time_distance_array, too_late_detect_objects,
+            misjudge_objects);
```

Comment:
Do you really need to declare a new variable?
```
        const std::string evasive_diag = generateEgoRiskEvasiveDiagnosis(
            *path, intersection_stoplines.closest_idx, time_distance_array,
            too_late_detect_objects, misjudge_objects);
```

Reply:
I couldn't think of it. Thank you.